### DISCUSSION TITLE
@gmap/v3 docs

### DISCUSSION THREAD

**Discussion by surgiie** [diegoazh/gmap-vue#300] [Q&A]
Is there a link to any docs for the newer vue 3 components? 

The docs here dont seem accurate or outdated:

https://diegoazh.github.io/gmap-vue/#about-gmapvue

**Comment by diegoazh** [comment]
Hi @surgiie, a few days before we deployed our new documentation site, we are working on it but, it has the main point to start using the plugin. You can use the same [link](https://diegoazh.github.io/gmap-vue).
